release-engineering/role-handbooks/release-image-promotion.md

```diff
@@ -26,10 +26,11 @@ cip-mm --base_dir=$GOPATH/src/k8s.io/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s
 cip-mm --base_dir=$GOPATH/src/k8s.io/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.19.1-rc.0
 ```
 
-_note_: If you clone the repo in another structure like `$GOPATH/src/github.com/kubernetes/k8s.io` you will nee to set the `--base_dir` as `--base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io`
+_note_: If you clone the repo in another structure like `$GOPATH/src/github.com/kubernetes/k8s.io` you will nee to set the `--base_dir` as `--base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io`  
```

Comment:
```suggestion
_note_: If you clone the repo in another structure like `$GOPATH/src/github.com/kubernetes/k8s.io` you will nee to set the `--base_dir` as `--base_dir=$GOPATH/src/github.com/kubernetes/k8s.io/k8s.gcr.io`
```

Reply:
I've intentionally added two spaces to move the second note to another row, as I thought it would look better, but I can remove the spaces.

Reply:
Did the newline not work? Maybe we can convert it to a list of "Notes"?

Reply:
This note doesn't exist anymore.